src/client/components/board.js

```diff
@@ -6,12 +6,18 @@ import Terminal from "./terminal";
 import Table from "./table";
 import PerfBudget from "./perf-budget";
 
+import Favicon from "react-favicon";
+
 import { readableBytes } from "../helpers/utils";
 import Nav from "./nav";
 
 import io from "socket.io-client";
```

Comment:
What is this

Reply:
The Favicon component i used. I don't get the question.

Reply:
hey @Remoc22 ! thank you for looking after the PRs! can we please try to provide more context while asking the developer why did they do something or about their approach? many thanks!